src/index.html

```diff
@@ -4,14 +4,182 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+        <label class="input-line">
+          Surname:
+          <input
+            name="surname"
+  autocomplete="off"
```

Comment:
<img width="1064" alt="Screenshot 2023-04-06 at 17 48 52" src="https://user-images.githubusercontent.com/53228259/230416152-fa358dd6-a1fc-4bb1-b856-274d252ecffb.png">
check all tags
```suggestion
          <input
            name="surname"
            autocomplete="off"
```